CHANGES.rst

```diff
@@ -6,8 +6,8 @@ CHANGES
 1.4.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
-
+- Make sure that `ImmutableContainer` does not accept transient objects.
+  That just breaks the world.
```

Comment:
Remove the last sentence and add this instead. "This is particularly important since objects can be initialized in transient state when not using the `create()` context manager."